### DISCUSSION TITLE
SQLite implementations

### DISCUSSION THREAD

**Discussion by johnbcodes** [serverlesstechnology/cqrs#34] [Show and tell]
I created SQLite forks/implementations of `postgres-es` [here](https://crates.io/crates/sqlite-es) and the `cqrs-demo` [here](https://github.com/johnbcodes/cqrs-es-demo-sqlite). Fairly trivial thanks to @davegarred and SQLx.

I [also explored](https://github.com/johnbcodes/cqrs-sqlite-sea-orm) using [sea-orm](https://www.sea-ql.org/SeaORM/) for `Query`s. This lead to a question/discussion [regarding error handling](https://github.com/serverlesstechnology/cqrs/discussions/33).

Thanks to @serverlesstechnology and @davegarred for such useful and well-document libraries!

**Comment by davegarred** [comment]
Thanks for the work @johnbcodes! I'll try to take a closer look next week (new job so time is a little tight right now).
